docs/5.x/changelog.md

```diff
@@ -21,6 +21,22 @@ LTS starts with `3.51.0` with minor backwards compatible changes added over time
 
 ## 5.x Releases
 
+### 5.2.0-rc.3
+
+#### Bugfixes
+
+* Fix an issue with deploying AWS clusters using provisioner.
+* Fix an issue with installers downloaded from the distribution portal.
+* Fix an issue with expanding clusters installed via an Ops Center.
+* Fix an issue with garbage collection not working on older clusters sometimes.
```

Comment:
This gives an impression of garbage collection being in GA for older versions while all this is to provide limited support in terms of a custom script file that uses low-level undocumented commands.
I would not mention it.

Reply:
Ok, I'll remove.